src/sync.js

```diff
@@ -513,7 +513,11 @@ SyncApi.prototype._syncFromCache = async function(savedSync) {
     debuglog("sync(): not doing HTTP hit, instead returning stored /sync data");
 
     // No previous sync, set old token to null
-    const oldSyncToken = null;
+    const syncEventData = {
+        oldSyncToken: null,
+        nextSyncToken,
```

Comment:
nextSyncToken isn't even set here

Reply:
[I feel like you could have spotted this yourself with a little more care...]

Reply:
oops, sorry